pkg/api/v1/status/atlasproject.go

```diff
@@ -72,4 +73,43 @@ type AtlasProjectStatus struct {
 
 	// The list of private endpoints configured for current project
 	PrivateEndpoints []ProjectPrivateEndpoint `json:"privateEndpoints,omitempty"`
+
+	// AuthModes contains a list of configured authentication modes
+	// "SCRAM" is default authentication method and requires a password for each user
+	// "X509" signifies that self-managed X.509 authentication is configured
+	AuthModes []authmode.AuthMode `json:"AuthModes,omitempty"`
```

Comment:
Added an AuthModes list to the Status.
Just realised that I need to put the default `SCRAM` somewhere, even though the field is not used at this stage.

Reply:
Also noticed that this status field isn't passed properly, fixing...

Reply:
I manually add "SCRAM" to the status. I is never removed cause there is no way to disable default auth through the operator.